online-shop/src/app/auth/login-display/login-display.component.html

```diff
@@ -0,0 +1,13 @@
+<form (ngSubmit)="onParentSubmit()" [formGroup]="formGroup">
+  <label for="username">Username: </label>
+  <input id="username" type="text" formControlName="username" />
+
+  <label for="password">Password: </label>
+  <input id="password" type="text" formControlName="password" />
```

Comment:
check out what other types of inputs are out there and pls use a more appropriate one for the password